src/components/fields/derived.js

```diff
@@ -539,16 +539,12 @@ export const HoverInfo = connectToContainer(UnconnectedFlaglist, {
       ].includes(container.type)
     ) {
       options.push({label: _('Z'), value: 'z'});
-    }
-
-    if (container.type === 'choropleth') {
+    } else if (container.type === 'choropleth') {
```

Comment:
did you change this to else if because the same trace type was present in more than one conditional?